docs/Catalog/RemovingStacks.md

```diff
@@ -0,0 +1,61 @@
+<!-- markdownlint-disable-next-line first-line-h1 -->
```

Comment:
We should probably include separate instructions for removing a stack managed with Terraform. In that case, Terraform is responsible for deleting the search cluster and database. Terraform also initiates the CloudFormation deletion.

I don't know if our terraform code will empty the analytics or cloudtrail buckets, or if those have been removed from the stack in the terraform variants.

Reply:
Clarified what happens. TIcked filed on IAC: https://github.com/quiltdata/iac/issues/70